packages/rpc/src/navie.ts

```diff
@@ -16,5 +16,19 @@ export namespace NavieRpc {
         commands: Command[];
       };
     }
+
+    export namespace Suggest {
+      export const Method = 'v1.navie.suggest';
+      export type NextStep = {
+        command: 'generate' | 'diagram' | 'plan' | 'test' | 'explain' | 'help';
+        prompt: string;
+        label: string;
+        overallScore: number;
+      };
+      export type Params = {
+        threadId: string;
```

Comment:
Would this be on the most recent assistant id or user message id, to get a suggestion from a specific point in the conversation?

Reply:
Currently this API reflects the capabilities of the UI, where next steps are only displayed for the most recent assistant response. It won't be difficult to transition to a state where we can request next steps from any point in the future if need be.